providers/implementations/asymciphers/rsa.c

```diff
@@ -0,0 +1,387 @@
+/*
```

Comment:
Mind, the next `rsa.c` will clash. `rsa_enc.c`?

Reply:
This is a remaining concern

Reply:
Fixed